omegaconf/_utils.py

```diff
@@ -4,10 +4,11 @@
 import string
 import sys
 from enum import Enum
-from typing import Any, Dict, List, Match, Optional, Tuple, Type, Union
+from typing import Any, Dict, List, Optional, Tuple, Type, Union
```

Comment:
Don't pay attention to the import changes in this commit, what happened is I just made one commit containing all import changes to get them "out of the way" of the more interesting changes found in subsequent commits.